tests/test/storage/test_shared_files.cpp

```diff
@@ -1,35 +1,19 @@
 #include <catch2/catch.hpp>
 
-#include "faabric/util/func.h"
 #include "faasm_fixtures.h"
 #include "utils.h"
 
 #include <boost/filesystem.hpp>
 
 #include <conf/FaasmConfig.h>
 #include <faabric/util/files.h>
+#include <faabric/util/func.h>
 #include <storage/FileLoader.h>
-#include <storage/SharedFiles.h>
 
 using namespace storage;
 
 namespace tests {
 
-class SharedFilesTestFixture : public S3TestFixture
-{
-  public:
-    SharedFilesTestFixture()
-      : loader(storage::getFileLoader())
-    {
-        SharedFiles::clear();
-    }
-
-    ~SharedFilesTestFixture() { SharedFiles::clear(); }
-
-  protected:
-    storage::FileLoader& loader;
-};
```

Comment:
As discussed in previous PR, this part of the system is now involved in tests in more than one file, so can be promoted into `faasm_fixtures.h`.